bootstrap/015_warehouse_views.sql

```diff
@@ -19,20 +19,20 @@ EVENTS AS (
             when event_name = 'RESUME_WAREHOUSE' and event_state = 'STARTED' then 1
             when event_name = 'RESUME_CLUSTER' and event_state = 'COMPLETED' then 1
             when event_name = 'SUSPEND_CLUSTER' and event_state = 'COMPLETED' then 2
-            when event_name = 'SUSPEND_WAREHOUSE' and event_state = 'COMPLETED' then 2
```

Comment:
Correct. It's annoying because there is no consistent way to backfill. Through some complex maneuvering you could likely correct anything that still exists in warehouse event history but it is a fairly expensive operation. Unfortunately, the older data there is no longer available so it means it would only address part of the bad data.

Reply:
yeah figured as much. Just wanted to call it out